src/action-creators/logout.js

```diff
@@ -25,17 +26,19 @@ export const logout = () => {
   localStorage.autologin = false
 
   // clear initial settings
-  importText([{ value: EM_TOKEN, rank: 0 }], INITIAL_SETTINGS)
+  await dispatch(importText([{ value: EM_TOKEN, rank: 0 }], INITIAL_SETTINGS))
```

Comment:
Careful! Adding `await` changes the behavior. `logout` should *not* wait for `importText` to complete. It can happen in the background.

Reply:
Ahh I thought it needed to wait. I will change it.